cli/cmd/plugin/ui/web/tanzu-ui/src/index.scss

```diff
@@ -17,6 +17,29 @@ code {
     font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+// override clarity computed line height of 20px
+[cds-text*=body], [cds-text*=subsection] {
+    line-height: 24px;
+}
+
+// custom elements
+.code {
+    vertical-align: middle;
+    color: white;
+    background-color: #222222;
+    font-family: inline;
+    font-family: "Fira Code", Menlo, Monaco, "Courier New", monospace;
```

Comment:
why do we set font-family here? Is it a special font-family for code class? If not, I suggest to add the font-family to body or a container in app level so that all the element can inherit the same font-family.

Reply:
Yes, this is specific to the "code" view for the CLI command equivalents